src/rootReducer.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+import appReducer from './components/App/reducer';
```

Comment:
Looks all good Zozo :)
The only minor thing I would like to let you know: don't use the name _Reducer_ in the state. If you import the reducer with the name of what it represents, your state will look nicer ;)

Example:

```
import app from './components/App/reducer';
import login from './components/Login/reducer';

export default combineReducers({
  app,
  login
});
```

This way, in the other components where you use state, it just reads nicer - e.g.

````
isLoggedIn =  useSelector(state => state.loginReducer.loggedIn);
```
becomes
```
isLoggedIn =  useSelector(state => state.login.loggedIn);
```

So, you did veeeery good! 👍 👍 👍

Reply:
Thank you @haensl 
I totally agree with your recommendation
Also I will add more stuff, to practice!